testing/gcin/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Contributor: Felix Yan <felixonmars@archlinux.org>
```

Comment:
Please use only tabs for indentation (and alignment) in APKBUILDs and other shell scripts, not spaces.